src/include/ceph_fs.h

```diff
@@ -654,6 +654,8 @@ struct ceph_mds_request_head {
 
 	__le32 ext_num_retry;          /* new count retry attempts */
 	__le32 ext_num_fwd;            /* new count fwd attempts */
+
+	__le32 owner_uid, owner_gid;   /* used for OPs which create inodes */
```

Comment:
We'd better initialize them to `-1` because `0` is still a validate value here.

Reply:
Have done